my-dsl.md

```diff
@@ -2,37 +2,102 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+[JsonDSL](https://github.com/lift/lift/tree/master/framework/lift-base/lift-json) 
+(part of the Liftweb framework for Scala)
 
 # Domain
 _Describe the language's domain in five words._
 
+Building name/value data structures
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+The code with the JSON's information goes through a parser that builds the JSON. 
+The program creates key/value pairs or concatenates when it sees parentheses or 
+~'s. Within each pair, implicit methods convert typical Scala types (i.e. Int, 
+String) into JSON types (i.e. JInt, JString). Finally, we get a JSON object 
+containing the name/value pairs.
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+This DSL is purely domain-specific because it has very limited capabilities. 
```

Comment:
This definitely looks like a DSL because of the limited capabilities, like you said. It would also be interesting to think about what we talked about in class. Would developers want to add more features to it? It seems like they have done some updating (it mentions migrating from version 2.2). Most of these seem pretty focused on things that will be helpful for its purpose, but soon, with enough changes, it could start supporting conditionals and loops. Would they do that? Would that sully the language?